### PR TITLE
Fix flake8 errors

### PR DESCRIPTION
**Pull request by wxmerkt**
Should fix the failing format job on gepgitlab

### CODE REVIEW

File: benchmarks/bench-switch.py [hunk 1]
```diff
@@ -3,8 +3,8 @@
 import eigenpy
 import numpy as np
 
-import time
-import timeit
+import time  # noqa
+import timeit  # noqa
```
Comment (nim65s) [inline benchmarks/bench-switch.py:7]: Couldn't we remove those imports ?

**Review by nim65s — APPROVED** [review]
Thanks for this !
I have a minor comment, and removed E203, as this is incompatible with black

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,2 @@
+[flake8]
+extend-ignore = E203
```
Comment (wxmerkt) [inline setup.cfg:2]: The gepetto linter config has linelength=119 and exclude cmake/, which this file would omit. Should we add it here as well?

File: benchmarks/bench-switch.py (hunk 1 above)
Comment (wxmerkt) [reply benchmarks/bench-switch.py:7]: When the benchmark is run with IPython it is being used but the linter cannot figure this out

File: setup.cfg (hunk 2 above)
Comment (nim65s) [reply setup.cfg:2]: Good catch, i'll add this extend ignore to gepetto linters. But the linelength will have to move to 88 for black :/

**Comment by nim65s** [comment]
now yapf is complaining, but we should abandon yapf for black now :(

File: setup.cfg (hunk 2 above)
Comment (nim65s) [reply setup.cfg:2]: PS: excluding cmake is not mandatory anymore for flake, since https://github.com/jrl-umi3218/jrl-cmakemodules/pull/524

**Comment by nim65s** [comment]
Actually, the format is already good for black, thanks to pre-commit, so we just need to tell that to gepetto-linters with a link in the readme :)

**Review by jcarpent — COMMENTED** [review]
Thanks @wxmerkt for your contribution and thanks @nim65s for quick review.

**Pull request merged**